### ISSUE TITLE
Elastisearch - Awaiting review records discrepancey

### ISSUE THREAD

**Issue by MNSmith** [BiologicalRecordsCentre/iRecord#594]
Looking at the list of records "Awaiting review" for aculeate Hymenoptera, if I use the new Elastisearch grid I can see 10 imported iNaturalist records belonging to a Recorder named "wildjohnlee" - eg 9919356 and 9924296.  If I do the same search on the old Verification Grid page these records do not appear.

As noted on #592, the only way to currently redet a record via the Elastisearch screen is through the "Edit this record" button.  If I try to do this for these "wildjohnlee" records I get the following message:

"Page load failed: The record you are trying to load does not exist."

**Comment by DavidHepper** [comment]
Bill Budd reports that he now has 52 records Unverified according to old Verify, but 67 Pending according to Verify - new.

**Comment by kitenetter** [comment]
The "wildjohnlee" records look like they have been removed - we discovered that the export from iNat was including records where the original recorder had retained their copyright, so these have had to be removed from the iRecord database. It appears that it is taking time for the deletes to feed through to the new verification page.

@johnvanbreda is this just a caching problem or does there need to be a specific refresh to remove such records from the elasticsearch view?
